lib/utils.sh

```diff
@@ -1,6 +1,13 @@
 #!/usr/bin/env bash
 
-RUBY_BUILD_VERSION="${ASDF_RUBY_BUILD_VERSION:-v20221121}"
+RUBY_BUILD_VERSION="${ASDF_RUBY_BUILD_VERSION:-}"
+if [ "$RUBY_BUILD_VERSION" = "" ]; then
+  RUBY_BUILD_VERSION="$(
+    curl --silent "https://api.github.com/repos/rbenv/ruby-build/releases/latest" |
```

Comment:
curl is a default dependency of asdf.
https://asdf-vm.com/guide/getting-started.html#_1-install-dependencies